### PR TITLE
Include comment anchors in comment event mails

### LINKED ISSUE #14494: emails about new comments lack the comment id
Each email about a new comment that has been added to a project or package must contain the comment id. That will simplify removal of comments. OBS is not a chat system after all, see comment `#1392411`.

Ideally the comment id should be part of the copy&paste command, like it is provided for new submit requests.

`osc comment delete 1787150`

### PR DESCRIPTION
**Pull request by hennevogel**
Let's send people to the right place directly

Fixes #14494

### CODE REVIEW

File: src/api/app/views/event_mailer/comment_for_package.html.haml [hunk 1]
```diff
@@ -1,6 +1,7 @@
-%p
-  Visit #{link_to(nil, package_show_url(event['project'], event['package'], only_path: false, host: @host))}
 %p
   = event['commenter'].realname
   wrote in package #{event['project']}/#{event['package']}:
-%div= render_as_markdown(event['comment_body'])
+%div
+  = render_as_markdown(event['comment_body'])
+%p
+  = link_to("View Comment", package_show_url(event['project'], event['package'], anchor: "comment-#{event['id']}", only_path: false, host: @host))
```
Comment (bmwiedemann) [inline src/api/app/views/event_mailer/comment_for_package.html.haml:7]:
```suggestion
  = link_to('View Comment', package_show_url(event['project'], event['package'], anchor: "comment-#{event['id']}", only_path: false, host: @host))
```

File: src/api/app/views/event_mailer/comment_for_project.html.haml [hunk 2]
```diff
@@ -1,6 +1,7 @@
-%p
-  Visit #{link_to(nil, project_show_url(event['project'], only_path: false, host: @host))}
 %p
   = event['commenter'].realname
   wrote in project #{event['project']}:
-%div= render_as_markdown(event['comment_body'])
+%div
+  = render_as_markdown(event['comment_body'])
+%p
+  = link_to("View Comment", project_show_url(event['project'], anchor: "comment-#{event['id']}", only_path: false, host: @host))
```
Comment (bmwiedemann) [inline src/api/app/views/event_mailer/comment_for_project.html.haml:7]:
```suggestion
  = link_to('View Comment', project_show_url(event['project'], anchor: "comment-#{event['id']}", only_path: false, host: @host))
```

File: src/api/app/views/event_mailer/comment_for_request.html.haml [hunk 3]
```diff
@@ -1,6 +1,7 @@
-%p
-  Visit #{link_to(nil, request_show_url(event['number'], only_path: false, host: @host))}
 %p
   = event['commenter'].realname
   wrote in request #{event['number']}:
-%div= render_as_markdown(event['comment_body'])
+%div
+  = render_as_markdown(event['comment_body'])
+%p
+  = link_to("View Comment", request_show_url(event['number'], anchor: "comment-#{event['id']}", only_path: false, host: @host))
```
Comment (bmwiedemann) [inline src/api/app/views/event_mailer/comment_for_request.html.haml:7]:
```suggestion
  = link_to('View Comment', request_show_url(event['number'], anchor: "comment-#{event['id']}", only_path: false, host: @host))
```

**Review by bmwiedemann — COMMENTED** [review]
rubocop/linter prefers single quotes for constant string literals

**Comment by bmwiedemann** [comment]
also: "Files should end with a trailing newline" for `comment_for_request.html.haml` and `comment_for_project.html.haml`

File: src/api/spec/mailers/event_mailer_spec.rb [hunk 4]
```diff
@@ -66,6 +66,7 @@
       let(:mail) { EventMailer.with(subscribers: Event::CommentForProject.last.subscribers, event: Event::CommentForProject.last).notification_email.deliver_now }
 
       it 'gets delivered' do
+        binding.pry
```
Comment (danidoni) [inline src/api/spec/mailers/event_mailer_spec.rb:69]:
```suggestion
```

**Pull request merged**